Add July 2 Standard community call (#91)
publiccodenet/blog: _posts/2020-07-20-notes-from-community-call-2-july-2020.md

---
title: "Notes from community call - 2 July 2020"
date: 2020-07-20
author: Jan Ainali
type: blogpost

> excerpt: In which we split Welcome contributions

Reviewer: ```suggestion
excerpt: Splitting 'Welcome contributions' in the Standard for Public Code
```

categories:
  - Community call
---

# 2 July 2020 Standard for Public Code community call

## Update from the Foundation

* The first Standard compliance analysis [has been merged](https://github.com/open-zaak/open-zaak/pull/609)!
* There's [a mailing list for the Standard for Public Code](https://lists.publiccode.net/mailman/postorius/lists/standard.lists.publiccode.net/).

## Attendees

* [Jan Ainali](https://publiccode.net/team/jan-ainali.html) (chair)
* [Eric Herman](https://publiccode.net/team/eric-herman.html)
* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
* [Alba Roza](https://publiccode.net/team/alba-roza.html)
* [Elena Findley-de Regt](https://publiccode.net/team/elena-findley-de-regt.html)
* [Laura Scheske](https://publiccode.net/team/laura-scheske.html)
* [Jonathan Pichot](https://twitter.com/_pichot)
* [Peter Kuhn](https://twitter.com/nhuKreteP)

## Notes

### Introductions

> Jonathan and Peter was given an introduction to the Foundation for Public Code and an explanation of codebase stewardship. Rather than attempt to go into details of the Standard for Public Code, they each decided to come back to a Foundation for Public Code community call in the future.

Reviewer: ```suggestion
Jonathan and Peter were given an introduction to the Foundation for Public Code and an explanation of codebase stewardship. Rather than attempt to go into details of the Standard for Public Code, they each decided to come back to a Foundation for Public Code community call in the future.
```

> ### Splitting Welcome contributions

Reviewer: ```suggestion
### Splitting the Standard for Public Code's [Welcome contributions](https://standard.publiccode.net/criteria/open-to-contributions.html) criterion
```

> Eric explained the background which is mostly captured in the notes from our last community call. Then [the draft](https://hackmd.io/R0Q4PeDoSSqaUPYt1uDaMA) was shown to everybody and a few minutes were given to just read it through. As a quick improvment the name for the new criterion *Supporting easy contributions* was changed to *Make contributing easy*.

Reviewer: ```suggestion
Eric explained the background which is mostly captured in the [notes from our last community call](https://blog.publiccode.net/community%20call/2020/06/22/notes-from-community-call-3-june-2020.html). Then [the draft](https://hackmd.io/R0Q4PeDoSSqaUPYt1uDaMA) was shown to everybody and a few minutes were given to just read it through. As a quick improvment the name for the new criterion *Supporting easy contributions* was changed to *Make contributing easy*.
```

We discussed the way the draft had split up the criterion and how governance played in with that. Jan and Elena explained that the thinking behind it was that Welcome Contributions is what you need to build trust, pique interest, provide legibility of leadership and the community goals. In contrast, Make contributing easy is supposed to cover the tools and processes that allows for contributors to find a way into the codebase.

We also agreed on that having a separate governance criterion could be somewhat of a trap. People will tend to build governance in the way that they know how to build governance structures, which is *closed*. Governance should not be a goal - there shouldn't be more governance than needed - and since we do recognize some is needed, those requirements should remain but not be given more weight.

With the explaination of the way the criterion was split the group found the requirement about the roadmap to fit better under Welcome contributions.

After changing the header and moving that requirement the consensus was that this was ready to become a [pull request](https://github.com/publiccodenet/standard/pull/347).

### Create reusable and portable code

> With a few minutes left on the call Eric raised the question that the requirement about reuse in the criterion [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html) might be hard to meet by codebases developed for state and suprastate purposes. He suggested "The codebase MUST be in use by multiple parties." should change the MUST to a SHOULD. Unfortunately there was not time for a deeper discussion during the call so it was decided to continue it in a [pull request](https://github.com/publiccodenet/standard/pull/346).

Reviewer: ```suggestion
With a few minutes left on the call Eric raised the question that the requirement about reuse in the criterion [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html) might be hard to meet by codebases developed for state and suprastate purposes. He suggested "The codebase MUST be in use by multiple parties." should be changed to a SHOULD. Unfortunately there was no time for a deeper discussion during the call so it was decided to continue it in a [pull request](https://github.com/publiccodenet/standard/pull/346).
```
